Bottom: e4cfc8fae1f33d18b1320ea10e2df4b9cefa3a57
Top:    1de42d8c251e13f6ce879bd8859f3ff9ce82d2c5
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 15:33:34 -0500

Install buildbot plugin dependencies


---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 0c67da56..e8005b9e 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -15,7 +15,11 @@ RUN apk --no-cache add \
 
 # install pip dependencies
 RUN pip --no-cache-dir install \
-    'attrs'
+    'attrs' \
+    'buildbot-console-view' \
+    'buildbot-grid-view' \
+    'buildbot-waterfall-view' \
+    'buildbot-www'
 
 # Create buildbot user
 ARG BUILDBOT_UID=1000
